app/controllers/overrides/registrations_controller.rb

```diff
@@ -4,19 +4,20 @@ class RegistrationsController < DeviseTokenAuth::RegistrationsController
     # method copied from original gem; refactored to please Rubocop
     def create
       setup_resource_for_create
+      return render_create_error if name_fields_have_invalid_characters?
+
       if resource_class.devise_modules.include?(:confirmable) && !@redirect_url
         return render_create_error_missing_confirm_success_url
       end
 
       # if whitelist is set, validate redirect_url against whitelist
-      if DeviseTokenAuth.redirect_whitelist
-        unless DeviseTokenAuth.redirect_whitelist.include?(@redirect_url)
-          return render_create_error_redirect_url_not_allowed
-        end
+      if DeviseTokenAuth.redirect_whitelist && !DeviseTokenAuth.redirect_whitelist.include?(@redirect_url)
```

Comment:
This is just a result of Ruby auto formatting

Reply:
note: This formatting is causing the diff coverage check to complain, we can ignore.